src/utility/constants/Status.js

```diff
@@ -0,0 +1,3 @@
+export const status = {
```

Comment:
don't create a separate file for this. 
Create a descriptive naming for this enum and export it from the same common component.

Reply:
ok